chia-tools/Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = "https://github.com/Chia-Network/chia_rs/chia-tools"
 chia-protocol = { version = "0.2.14", path = "../chia-protocol" }
 chia-traits = { path = "../chia-traits" }
 clvm-utils = { path = "../clvm-utils" }
-clvm-traits = { path = "../clvm-traits" }
+clvm-traits = { version = "=0.2.14" }
```

Comment:
nit: this can just be `clvm-traits = "=0.2.14"`. same for chia-wallet, chia-wallet/fuzz, clvm-utils, clvm-utils/fuzz, fuzz, and anywhere else

Reply:
true